coffea/jetmet_tools/CorrectedJetsFactory.py

```diff
@@ -70,9 +70,10 @@ def jer_smear(
 
     jersmear = jet_energy_resolution * jet_resolution_rand_gauss
     jersf = jet_energy_resolution_scale_factor[:, variation]
-    doHybrid = pt_gen > 0
+    deltaPtRel = (jetPt - pt_gen) / jetPt
+    doHybrid = (pt_gen > 0) & (np.abs(deltaPtRel) < 3)
 
-    detSmear = 1 + (jersf - 1) * (jetPt - pt_gen) / jetPt  # because of awkward1.0#367
+    detSmear = 1 + (jersf - 1) * deltaPtRel  # because of awkward1.0#367
```

Comment:
https://github.com/scikit-hep/awkward-1.0/issues/367 seems to be about datetime, what is the comment about?
The code as written seems straightforward, i.e. not a workaround.

Reply:
yeah stale comment.